package.json

```diff
@@ -46,7 +47,7 @@
     "regenerator-runtime": "^0.13.7",
     "text-encoding": "^0.7.0",
     "typescript": "^4.3.5",
-    "vite": "^2.6.14",
+    "vite": "^2.9.1",
```

Comment:
Updating Vite to include fix from https://github.com/vitejs/vite/pull/7098 which allows `import workerPath from 'hydrogen-view-sdk/main.js?url';` to still work even though we have `package.json` `exports` defined.

Reply:
Re-commented on the diff since this one is outdated, https://github.com/vector-im/hydrogen-web/pull/693#discussion_r866272963